service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -142,6 +143,7 @@ public SubmittedCallbackResponse handleSubmittedEvent(@RequestBody CallbackReque
 
         if (caseDataBefore.getState() == OPEN) {
             publishEvent(new SubmittedCaseEvent(caseData, caseDataBefore));
+            publishEvent(new RespondentsUpdated(caseData, caseDataBefore));
```

Comment:
clarify if this needs to be sent in returned also?

Reply:
need to check with business/qa